### PR TITLE
fix: i18n desc of broker_shared_subscription_strategy

### PR DESCRIPTION
**Pull request by qzhuyan**
Fix: EMQX-9007

I just fetched the description from old commits. 

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

~~Added tests for the changes~~
~~Changed lines covered in coverage report~~
- [x] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` and `.zh.md` files
- [x] For internal contributor: there is a jira ticket to track this change
~~If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up~~
~~Schema changes are backward compatible~~

### CODE REVIEW

**Review by id — APPROVED** [review]

**Review by lafirest — APPROVED** [review]

**Review by thalesmg — APPROVED** [review]

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 1]
```diff
@@ -1070,14 +1070,20 @@ Supported configurations are the following:
...
   - `sticky`: always use the last selected subscriber to dispatch, until the subscriber disconnects.
-  - `hash`: select the subscribers by the hash of `clientIds`"""
+  - `hash_clientid`: select the subscribers by hashing the `clientIds`
+  - `hash_topic`: select the subscribers by hashing the source topic"""
 
             zh: """共享订阅消息派发策略。
   - `random`：随机挑选一个共享订阅者派发；
   - `round_robin`：使用 round-robin 策略派发；
+  - `round_robin_per_group`: 在共享组内循环选择下一个成员；
+  - `local`: 选择随机的本地成员，否则选择随机的集群范围内成员;
```
Comment (HJianBo) [inline apps/emqx/i18n/emqx_schema_i18n.conf:1083]: nit: `：`

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 2]
```diff
@@ -1070,14 +1070,20 @@ Supported configurations are the following:
...
+  - `local`: select random local subscriber otherwise select random cluster-wide
   - `sticky`: always use the last selected subscriber to dispatch, until the subscriber disconnects.
-  - `hash`: select the subscribers by the hash of `clientIds`"""
+  - `hash_clientid`: select the subscribers by hashing the `clientIds`
+  - `hash_topic`: select the subscribers by hashing the source topic"""
 
             zh: """共享订阅消息派发策略。
   - `random`：随机挑选一个共享订阅者派发；
   - `round_robin`：使用 round-robin 策略派发；
+  - `round_robin_per_group`: 在共享组内循环选择下一个成员；
```
Comment (LenaLenaPan) [inline apps/emqx/i18n/emqx_schema_i18n.conf:1082]: 通过对发送者的客户端 ID 进行 Hash 处理来选择订阅者

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 3]
```diff
@@ -1070,14 +1070,20 @@ Supported configurations are the following:
...
+  - `local`: 选择随机的本地成员，否则选择随机的集群范围内成员;
   - `sticky`：总是使用上次选中的订阅者派发，直到它断开连接；
-  - `hash`：使用发送者的 Client ID 进行 Hash 来选择订阅者。"""
+  - `hash_clientid`：使用发送者的 Client ID 进行 Hash 来选择订阅者;
+  - `hash_topic`: 使用源主题进行 Hash 来选择订阅者。"""
```
Comment (LenaLenaPan) [inline apps/emqx/i18n/emqx_schema_i18n.conf:1086]: 通过对源主题进行 Hash 处理来选择订阅者

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 4]
```diff
@@ -1070,14 +1070,20 @@ Supported configurations are the following:
...
+  - `hash_topic`: select the subscribers by hashing the source topic"""
 
             zh: """共享订阅消息派发策略。
   - `random`：随机挑选一个共享订阅者派发；
   - `round_robin`：使用 round-robin 策略派发；
+  - `round_robin_per_group`: 在共享组内循环选择下一个成员；
+  - `local`: 选择随机的本地成员，否则选择随机的集群范围内成员;
   - `sticky`：总是使用上次选中的订阅者派发，直到它断开连接；
-  - `hash`：使用发送者的 Client ID 进行 Hash 来选择订阅者。"""
+  - `hash_clientid`：使用发送者的 Client ID 进行 Hash 来选择订阅者;
```
Comment (LenaLenaPan) [inline apps/emqx/i18n/emqx_schema_i18n.conf:1085]: 通过对发送者的客户端 ID 进行 Hash 处理来选择订阅者

**Review by LenaLenaPan — COMMENTED** [review]
add some comments

File: apps/emqx/i18n/emqx_schema_i18n.conf (hunk 4 above)
Comment (qzhuyan) [reply apps/emqx/i18n/emqx_schema_i18n.conf:1085]: comments are addressed in https://github.com/emqx/emqx/pull/10284

**Pull request merged**